client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/operations/ReplaceIfUnmodifiedOperation.java

```diff
@@ -35,6 +36,7 @@ protected VersionedOperationResponse executeOperation(Transport transport) {
       transport.writeArray(key);
       transport.writeVInt(lifespan);
       transport.writeVInt(maxIdle);
+      codec.writeExpirationNanoTimes(transport, lifespanNanos, maxIdleNanos, internalFlags);
```

Comment:
OK. I'll update the PR to change the semantic of the two fields based on a flag.

I've added new fields to avoid changing the type of lifespan/maxidle VInt->VLong (as toNanos(30days) doesn't fit in VInt) but I guess that's what new protocol versions are for. :-)

Thanks @galderz @wburns for the feedback!